commands/tier.mjs

```diff
@@ -12,17 +12,17 @@ const defaultFunction = {
         .setDescription('Shows the criteria for loot tiers'),
 
     async execute(interaction) {
-        const embed = new MessageEmbed();
-        const tiers = getTiers();
-        embed.setTitle("Loot Tiers");
-        embed.setDescription(`
-            Loot tiers are divided primarily by the per-slot value of the item:
-            • ${lootTier(tiers.legendary).msg} >= ${tiers.legendary.toLocaleString()}₽
-            • ${lootTier(0, true).msg}
-            • ${lootTier(tiers.great).msg} >= ${tiers.great.toLocaleString()}₽
-            • ${lootTier(tiers.average).msg} >= ${tiers.average.toLocaleString()}₽
-            • ${lootTier(tiers.average -1).msg} < ${tiers.average.toLocaleString()}₽
-        `);
+		const tiers = getTiers();
+        const embed = new MessageEmbed()
+        	.setTitle("Loot Tiers")
+        	.setDescription(`
+				Loot tiers are divided primarily by the per-slot value of the item:
+				• ${lootTier(tiers.legendary).msg} >= ${tiers.legendary.toLocaleString()}₽
+				• ${lootTier(0, true).msg}
+				• ${lootTier(tiers.great).msg} >= ${tiers.great.toLocaleString()}₽
+				• ${lootTier(tiers.average).msg} >= ${tiers.average.toLocaleString()}₽
+				• ${lootTier(tiers.average -1).msg} < ${tiers.average.toLocaleString()}₽
+			`);
```

Comment:
fmt went crazy?

Reply:
What do you mean by "fmt"? That code is correct and working very well for me.

Reply:
fmt = formatter.
This doesnt look correct for me especially line 15.